tests/test_parse.py

```diff
@@ -341,3 +341,86 @@ def test_disallow_abbrev(self, capsys):
             parse(self.Config, ["--so", "something_else"], allow_abbrev=False)
         captured = capsys.readouterr()
         assert "error: unrecognized arguments: --so something_else" in captured.err
+
+
+class SecretStr(str):
```

Comment:
Instead of wrapping the secret like this, wouldn't it be better to add an option that hides the default?